src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -373,7 +373,18 @@ def _create_gitrepo_volume_mount(gitrepo_volume, gitrepo_mount_path):
 def _create_ip_address(ip_address, ports, protocol, dns_name_label):
     """Create IP address. """
     if (ip_address and ip_address.lower() == 'public') or dns_name_label:
-        return IpAddress(ports=[Port(protocol=protocol, port=p) for p in ports], dns_name_label=dns_name_label)
+        return IpAddress(ports=[Port(protocol=protocol, port=p) for p in ports],
+                         dns_name_label=dns_name_label, type=ContainerGroupIpAddressType.public)
+
+
+def restart_container(client, resource_group_name, name, no_wait=False):
+    """Restart a running container group. """
+    return sdk_no_wait(no_wait, client.restart, resource_group_name, name)
+
+
+def stop_container(client, resource_group_name, name):
+    """Stop a running container group. """
+    return client.stop(resource_group_name, name)
```

Comment:
It seems like these could simply be reflected from the SDK instead of being custom commands.

Reply:
Do you know of an example anywhere on how to set up the command to just use reflection from the SDK?

Reply:
It's how most of the CLI is written.  If you search for `g.command(` you will get hundreds of examples :)

Reply:
There's nothing wrong with the approach though. It's just you don't get anything for free and have to maintain it all manually.